homeassistant/components/reolink/manifest.json

```diff
@@ -3,8 +3,9 @@
   "name": "Reolink IP NVR/camera",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/reolink",
-  "requirements": ["reolink-aio==0.2.1"],
+  "requirements": ["reolink-aio==0.2.2"],
```

Comment:
Can we bump the library in a separate PR with minimum amount of changes, just what's required to use the new library version?

Reply:
I made a sperate PR for the version bump here: https://github.com/home-assistant/core/pull/85848
I tried to do as little changes as possible, but some are needed to catch the new exceptions and changed return types.

Reply:
Thanks! That's merged now.